sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -0,0 +1,371 @@
+-- Query first observations for Firefox Android Clients.
+WITH first_seen AS (
+  SELECT
+    client_id,
+    first_seen_date
+  FROM
+    `mozdata.fenix.baseline_clients_first_seen`
+  WHERE
+    submission_date = @submission_date
+),
+-- Find earliest first_session ping per client.
+first_session_ping AS (
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(SAFE.PARSE_DATETIME('%F', SUBSTR(client_info.first_run_date, 1, 10))) AS first_run_datetime,
+    ARRAY_AGG(normalized_country_code ORDER BY metrics.datetime.first_session_timestamp ASC)[
```

Comment:
If the `first_session` ping is not reported, the dimensions (first_reported_country, first_reported_isp, channel, device_manufacturer, device_model, os_version) will be NULL.
To solve it, collect these dimensions from `mozdata.fenix.baseline_clients_first_seen` .